zebra-consensus/src/block/check.rs

```diff
@@ -154,7 +154,7 @@ pub fn subsidy_is_valid(block: &Block, network: Network) -> Result<(), BlockErro
         .activation_height(network)
         .expect("Canopy activation height is known");
 
-    if height < SLOW_START_INTERVAL {
+    if (height - SLOW_START_INTERVAL).is_none() {
```

Comment:
The original comparison seems clearer here, is there a reason to also reject heights over i32::MAX?

Reply:
155ce35d1d